doc/changes/changes-0.3.0.md

```diff
@@ -0,0 +1,35 @@
+# Integration-Test-Docker-Environment 0.3.0, released 2021-06-08
+
+In this release, we add support for new Exasol Docker-DB versions and Docker runtimes to the integration-test-docker-environment. For example, you can run the Docker-DB with the [NVIDIA Container Runtime](https://github.com/NVIDIA/nvidia-container-runtime) to add support for GPUs to the UDFs of the Exasol database. This allows you to test your GPU-accelerated UDFs in your CI environment, before deploying them on your production cluster. Furthermore, we converted the project from a pipenv project to a poetry project to improve packaging and reusability in the future.
+
+### New supported Exasol Versions
+
+* **6.2**: 6.2.14, 6.2.15,
+* **7.0**: 7.0.4, 7.0.5, 7.0.6, 7.0.7, 7.0.8, 7.0.9, 7.0.10
+
+If you need further versions, please open an issue.
+
+### Tested Docker Runtimes
+
+- Docker Default Runtime
+- [NVIDIA Container Runtime](https://github.com/NVIDIA/nvidia-container-runtime) for GPU accelerated UDFs
```

Comment:
Not sure if this is valid!!!
@tkilias  please confirm

Reply:
yes, it is valid. Nothing has changed on that and got once tested. CI is not yet possible. Tested doesn't mean that we support it.